test/unit/logic/referral.spec.ts

```diff
@@ -798,7 +798,7 @@ describe('referral-component', () => {
           expect(mockEmail.sendEmail).toHaveBeenCalledWith(
             'marketing@decentraland.org',
             '[Action Needed] IRL Swag Referral Tier Unlocked',
-            `<p>A user has unlocked the IRL Swag Referral Tier and provided the following email for contact: ${validEmail}</p>`
+            `A user has unlocked the IRL Swag Referral Tier and provided the following email for contact: ${validEmail}`
```

Comment:
Is this change expected? won't break email format?

Reply:
we are sending that tag as text in the emails instead of as a component